env_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"testing"
 
-	"github.com/caarlos0/env"
+	"github.com/dselans/env"
```

Comment:
Same as above:

I switched it while writing the patch (since I was working in my fork) - I can push a quick update to both.